hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/BucketManagerImpl.java

```diff
@@ -272,16 +272,6 @@ public void setBucketProperty(OmBucketArgs args) throws IOException {
           .setBucketName(oldBucketInfo.getBucketName());
       bucketInfoBuilder.addAllMetadata(args.getMetadata());
 
-      //Check ACLs to update
-      if (args.getAddAcls() != null || args.getRemoveAcls() != null) {
-        bucketInfoBuilder.setAcls(getUpdatedAclList(oldBucketInfo.getAcls(),
-            args.getRemoveAcls(), args.getAddAcls()));
-        LOG.debug("Updating ACLs for bucket: {} in volume: {}",
-            bucketName, volumeName);
-      } else {
-        bucketInfoBuilder.setAcls(oldBucketInfo.getAcls());
```

Comment:
We need to keep bucketInfoBuilder.setAcls(oldBucketInfo.getAcls()) otherwise existing acls will be lost upon setBucketProperty. Can you add a unit test to cover this?

Reply:
Thanks for catching fixed it, and also added a test to cover this scenario.